tests/integration/update_cluster/restrict_access/in-v1alpha2.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: "2016-12-10T22:42:27Z"
+  name: restrictaccess.example.com
+spec:
+  kubernetesApiAccess:
+  - 1.1.1.1/0
+  - 2001:0:85a3::/40
+  channel: stable
+  cloudProvider: aws
+  configBase: memfs://clusters.example.com/restrictaccess.example.com
+  etcdClusters:
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    name: main
+  - etcdMembers:
+    - instanceGroup: master-us-test-1a
+      name: us-test-1a
+    name: events
+  kubernetesVersion: v1.8.0
+  masterInternalName: api.internal.restrictaccess.example.com
+  masterPublicName: api.restrictaccess.example.com
+  networkCIDR: 172.20.0.0/16
+  networking:
+    kubenet: {}
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 1.1.1.1/0
+  - 2001:0:85a3::/40
```

Comment:
You probably could have just added this to `complex`, which is a sort of grab-bag for random things.  But this works :-)